applications/exam-play-project.md

```diff
@@ -0,0 +1,203 @@
+# EXAM PLAY PROJECT
+
+**NOT THIS IS A TEST APPLICATION, TO RUN THROUGH THE PROCCESS FOR THE CURATORS**
```

Comment:
**NOTE THIS IS A TEST APPLICATION, TO RUN THROUGH THE PROCESS FOR THE CURATORS

Reply:
Nice

Reply:
Fixed the typo in the commit